### PR TITLE
Update testing support with TestApp

### PR DESCRIPTION
**Pull request by emdobrin**
## Description

Add support for integration testing with Identity extension and custom ids 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/testapp/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -51,10 +51,10 @@
         android:layout_marginStart="56dp"
         android:layout_marginLeft="60dp"
         android:layout_marginTop="24dp"
-        android:onClick="trackQueue"
+        android:onClick="getQueueSize"
         android:text="Get Queue Size"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/setIdentifier" />
+        app:layout_constraintTop_toBottomOf="@+id/setVid" />
```
Comment (addb) [inline code/testapp/src/main/res/layout/activity_main.xml:57]: optout button id needs to be updated

current: L105 `app:layout_constraintTop_toBottomOf="@+id/unknown"`

**Review by addb — APPROVED** [review]
One nit comment. Otherwise LGTM!

File: code/testapp/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (emdobrin) [reply code/testapp/src/main/res/layout/activity_main.xml:57]: The button ID looks ok to me android:id="@+id/optOut", and the line you shared is the layout top constraint, which is also correct (under button unknown). Am I missing anything?

Comment (addb) [reply code/testapp/src/main/res/layout/activity_main.xml:57]: Screenshot 2023-02-17 at 11 53 57 AM

**Review by addb — APPROVED** [review]

**Pull request merged**